pkg/volume/local/local.go

```diff
@@ -204,8 +210,18 @@ func (m *localVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 		glog.Errorf("cannot validate mount point: %s %v", dir, err)
 		return err
 	}
-	if !notMnt {
```

Comment:
If we leave this check here then we cannot mount the same volume by another pod. As the original logic was to return nil if notMnt is false.